### ISSUE TITLE
Ligature request for Twig syntax

### ISSUE THREAD

**Issue by patkar** [tonsky/FiraCode#408]
Hi, can we have optimized ligatures for Twig syntax? The standard symbols are:

`{% some_instruction %}`  
`{{ some_variable }}`

and with whitespace control:

`{%- some_instruction -%}`  
`{{- some_variable -}}`

**Comment by dhouck** [comment]
Jinja2 and I think some other template systems use this also.

**Comment by patkar** [comment]
Yes, twig is basically jinja2 syntax. But I don't know if Jinja2 itself has also this whitespace control :)

**Comment by dhouck** [comment]
It does, which is nice. I don’t know Twig so I don’t know if it works exactly the same way, but that is how you would do whitespace control.

**Comment by dch** [comment]
NB Prolog and Erlang and possibly other languages have generic nested data structures that include `{{` for example: `{{a, nested, tuple}, inside, a tuple}` but `{[{ a, tuple}, {that, is}, {part, of}, {a, list}]}` so I'm not sure how you could manage these sorts of edge cases.

**Comment by dhouck** [comment]
Yeah, I was thinking of C++ now doing that too. But having tuples nested exactly two deep where the first element of the first tuple is negative is pretty rare. The `{{`  or `}}` character sequence might be incorrectly ligatured, but if it just displays with the characters closer together like `..`, `--`, `>>`, etc., the false positives won’t be that big a deal.

**Comment by dhouck** [comment]
I just realized that this is a duplicate of #244
